packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -80,39 +94,85 @@ const balanceOptions = [
   { name: "Max", divider: BalanceDivider.MAX },
 ];
 
+const customParseAsTimestamp = createParser({
```

Comment:
this function feels out place :/

Reply:
Where should we define it? It's only being used here so I feel it should be used here. Open to suggestions

Reply:
it's very related to `searchParams` so the only place I see fit is creating a `searchParams.ts` on utils folder.

But if you prefer I'm okay with letting it live here for now.

Reply:
Let's leave it here for now. If we need to use it on another file I would move it then.